db/sql/table05_pursuance_hierarchical_relationships.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE pursuance_hierarchical_relationships (
+  parent_id   integer NOT NULL REFERENCES pursuances ON DELETE CASCADE,
+  child_id    integer NOT NULL REFERENCES pursuances ON DELETE CASCADE,
+  PRIMARY KEY (parent_id, child_id)
+);
+ALTER TABLE pursuance_hierarchical_relationships OWNER TO superuser;
```

Comment:
@ajvb Just realized that I can easily imagine a parent-child pursuance relationship needing to become a peer-to-peer/bi-directional relationship... but that could be modeled with 2 hierarchical relationships, though, which would be really clean, so maybe this is perfect...